inst/schema/spec.md

```diff
@@ -0,0 +1,229 @@
+This is all highly uncertain and liable to change!
+
+# Data
+## POST /data/tab-name/
+Accepts a set of baseline options, and returns a data set for the data under the named tab.
+
+Request schema: [DataOptions.schema.json](./DataOptions.schema.json)
+
+Return schema: [Data.schema.json](./Data.schema.json)
+
+### Example
+#### Request
+```
+{
+  "irs_future": "80%",
+  "sprayInput": 1,
+  "biting_indoors": "high",
+  "population": 1000
+}
+```
+
+#### Response
+```
+[
+    {
+        "intervention": "none",
```

Comment:
So this is assuming a wide format, partly because that's how the current impact table looks, partly because it makes deriving the data series easy - you just tell the graph which columns should be used as the data series https://github.com/mrc-ide/mintr/pull/1/files#diff-2624be92b39903fa76764d78bef5f87bR142